.eslintrc.js

```diff
@@ -42,6 +42,9 @@ module.exports = {
     'prefer-template': 0,
     'no-underscore-dangle': 0,
     'no-negated-condition': 'error',
+    // TODO: re-enable once datasource class migration
+    'class-methods-use-this': 0,
```

Comment:
Sure:

`getReleases` in `clojure` doesn't use `this` as it uses `getReleases` from `maven` instead of using its own implementation.

`handleSpecificErrors` in the abstract class `Datasource` is a no-op, and not marked as abstract, as not all datasources may need to handle specific errors, so `handleSpecificErrors` is flagged.

I figured it was easier to disable this here during migration, and it can then be re-enabled for specific methods later? I can flag those specific methods now, if you think it's easier or better.

Reply:
i think it better to flag the causing code, otherwise you need a lot effort to fix it again

Reply:
Okay, removed this from here, and added it to each individual instance.